static/js/tools/map/fetcher/georaster.ts

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Fetch Geo Raster data.
+ */
+
+import geoblaze from "geoblaze";
+import _ from "lodash";
+import { Dispatch, useContext, useEffect } from "react";
+
+import { ChartDataType, ChartStoreAction } from "../chart_store";
+import { Context } from "../context";
+
+export function useFetchGeoRaster(dispatch: Dispatch<ChartStoreAction>): void {
+  const { placeInfo, statVar, display } = useContext(Context);
+  useEffect(() => {
+    const contextOk =
+      placeInfo.value.enclosingPlace.dcid &&
+      placeInfo.value.enclosedPlaceType &&
+      statVar.value.dcid &&
+      (!statVar.value.perCapita || statVar.value.denom) &&
```

Comment:
why checking statVar.value.denom here?

Reply:
ah, I forgot we're not doing per capita for geotiff for now. I was thinking that since we would fetch geotiff with per capita already calculated, we would need to pass the denom. Removed